dbt_project.yml

```diff
@@ -0,0 +1,13 @@
+name: 'dynamics_365_crm'
+version: '0.1.0'
+config-version: 2
+require-dbt-version: [">=1.3.0", "<2.0.0"]
+
+models:
+  dynamics_365_crm:
+    +materialized: table
```

Comment:
Open question - for Salesforce Formula Fields we materialize them as `view` to be as up to date as possible. Should we consider that here as well, or do you feel table is best for this use case?